RunJavaUtils.py

```diff
@@ -23,7 +23,7 @@ def Clean_And_Remove_Temp_Dir(tempDir):
     dirs = os.listdir(tempDir)
     for file in dirs:
         if (file != "." and file != ".."):            
```

Comment:
Not part of this diff, but is this logic necessary? The [Python docs](
https://docs.python.org/3/library/os.html#os.listdir) say, "The list is in arbitrary order, and does not include the special entries '.' and '..' even if they are present in the directory."